data_tables/Court_list_dynamoDB.tf

```
@@ -0,0 +1,24 @@
+resource "aws_dynamodb_table" "CourtList" {
+  name           = "CourtList"
+  billing_mode   = "PAY_PER_REQUEST"
+  hash_key       = "Date"
+  range_key      = "CourtNumber"
```

i would use lower case attributest
```suggestion
  range_key      = "courtNumber"
```